projects/debouncer/inc/debouncer.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
```

Comment:
Please add a module comment explaining the purpose of this module and basic usage.